common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2085,6 +2091,12 @@
     },
     "sqlState" : "22023"
   },
+  "INVALID_PARAMETRIZED_QUERY" : {
+    "message" : [
+      "Parametrize query must either use positional, or named parameters, but not both."
+    ],
+    "sqlState" : "42609"
```

Comment:
42609 is for situations like ? = ? where Db2 can't to late binding because it can't derive the type from context.
42613 seems most appropriate "clauses are mutually exclusive" (although parameter markers really aren't clauses...)